devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusListCategories.java

```diff
@@ -0,0 +1,54 @@
+package io.quarkus.gradle.tasks;
+
+import org.gradle.api.GradleException;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.Optional;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.options.Option;
+
+import io.quarkus.devtools.commands.ListCategories;
+import io.quarkus.devtools.project.QuarkusProject;
+
+public class QuarkusListCategories extends QuarkusPlatformTask {
+
+    private boolean fromCli = false;
+    private String format = "";
+
+    @Input
+    public boolean isFromCli() {
+        return fromCli;
+    }
+
+    @Option(description = "List only installed extensions.", option = "fromCli")
```

Comment:
Is that what the `fromCli` flag is for? I'm not sure I agree with this usage if it is.
My understanding is that `fromCli` should be used only to tag/indicate when the task invocation is wrapped by the CLI 

This could also just be a bad paste for the description...

Reply:
No, it's a wrong description. I copy pasted it from `QuarkusListExtensions`, it's wrong there too :).

AFAIK it's purely a way to indicate to lower layers if an execution comes from `quarkus` CLI (which delegates to maven/gradle) or real maven/gradle. `quarkus` CLI set this to true.